### ISSUE TITLE
update view names

### ISSUE THREAD

**Issue by LindsayYoung** [18F/openFEC#2042]
See attachment for updated names
[fec_views.xls.zip](https://github.com/18F/openFEC/files/595487/fec_views.xls.zip)

**Comment by LindsayYoung** [comment]
Talked to DBAs and they are going to drop warehouse tables we don't use, in dev and rename the views to the the names we are already using, so we won't need to make code changes.

We will make sure this works on dev and then update the other RDS databases.

Keeping this open for now to track progress.

**Comment by LindsayYoung** [comment]
This might be tricker than we thought. 

It looks like using the views directly might cause a problem:
```
psycopg2.ProgrammingError: "fec_vsum_sched_e" is a view
DETAIL:  Views cannot have row-level BEFORE or AFTER triggers.
```

Also, the new views don't have pg_date, that is the date they were loaded in the database, we could do a time stamp or eliminate this field. This is not as big of a deal

**Comment by jontours** [comment]
Yes, I ran into this same issue when trying to add triggers to the tables that supported the sched_e notice tables.  Basically I added  triggers to the pertinent supporting table for that view, pertinent table being the the one that has a foreign key reference to a parent table.

**Comment by LindsayYoung** [comment]
Maybe we can fix it with the "instead of" syntax?
https://www.postgresql.org/docs/9.1/static/sql-createtrigger.html

**Comment by LindsayYoung** [comment]
OK, I got that to run the prepare schedule e command, but get_actual_weekly_itemized_totals uses pg_date to pull recent reports, I am wondering if we should make the load date the time stamp it comes in to our system instead

**Comment by jontours** [comment]
Putting the list here as I'm not sure best place to put it, but it is related.  Currently I believe the warehouse tables that are being deprecated can be mapped to these current tables in Postgres:
Warehouse			Postgres
dimcmteproperties       disclosure.nml_form_1
dimreporttype		       public.ref_rpt_tp
dimparty	                       disclosure.nml_form_2
vw_filing_history            ?????

**Comment by LindsayYoung** [comment]
for the filing migration- https://github.com/18F/openFEC-web-app/issues/1371

**Comment by noahmanger** [comment]
**Criteria for completion:**
- [ ] Connect the views to the new tables

**Comment by LindsayYoung** [comment]
I started things here when I thought I had an easy fix, but did not get the updates going. 
https://github.com/18F/openFEC/pull/2052

**Comment by LindsayYoung** [comment]
I seem to be pretty close, but I am having an issue with is_notice, which is weird since that did not change

**Comment by noahmanger** [comment]
Next steps:
- Test in development to make sure it's working
- Coordinate the change across the different environments with DBAs

**Comment by noahmanger** [comment]
Is there a way that we could outsource the task of checking if this is working on develop? Is that an engineering task or could someone who's a careful proofer do it?

**Comment by LindsayYoung** [comment]
It is pretty straight forward to check, because you can see if the overnight updates work or not. I was working for a couple of days and it got over ridden by new changes to develop. 

Now there is a small issue merging in recent changes to get the branch up to speed. I nee to roll it back and try merging it again https://github.com/18F/openFEC/pull/2052

The biggest lift on this is the coordination that needs to happen with the FEC and diverting traffic while things get rebuilt.

**Comment by LindsayYoung** [comment]
closed with #2052
